src/pages/Search/AdvancedSearchFilters.tsx

```diff
@@ -213,13 +214,28 @@ const typeFiltersKeys: Record<string, Array<Array<ValueOf<typeof CONST.SEARCH.SY
 };
 
 function getFilterCardDisplayTitle(filters: Partial<SearchAdvancedFiltersForm>, cards: CardList) {
-    const filterValue = filters[CONST.SEARCH.SYNTAX_FILTER_KEYS.CARD_ID];
-    return filterValue
-        ? Object.values(cards)
-              .filter((card) => filterValue.includes(card.cardID.toString()))
-              .map((card) => getCardDescription(card.cardID, cards))
-              .join(', ')
-        : undefined;
+    const cardIdsFilter = filters[CONST.SEARCH.SYNTAX_FILTER_KEYS.CARD_ID] ?? [];
+    const feedFilter = filters[CONST.SEARCH.SYNTAX_FILTER_KEYS.FEED] ?? [];
+    const groupedCards: Record<string, WorkspaceCardsList> = {};
```

Comment:
It is hard to understand just from name `groupedCards` what is the purpose of this const. Can we come up with a better name?